### ISSUE TITLE
Sentinel scheduled tasks messages ignore config

### ISSUE THREAD

**Issue by dianabarsan** [medic/medic-webapp#4735]
Scheduled messages that do not use translation keys (and are generated on `registration`) ignore configuration.
Affected version: 2.15, 2.16, 3.0, 3.1.

**Comment by ngaruko** [comment]
Any tips on how to AT this @dianabarsan ?

**Comment by dianabarsan** [comment]
The way I discovered this bug was while writing e2e tests that would check that dates in scheduled messages were being i18ned (https://github.com/medic/medic-webapp/issues/4681) when I noticed that whole messages weren't being translated. 

So, you would need a schedule that doesn't use `translationKey`, instead use the old inline `messages` configuration. Then change your configuration so it's effects would be obvious. There are some options here:
- change outgoing messages language (`locale` or `locale_outgoing`) and have multiple languages listed in the `messages` property of your schedule, your scheduled task would have to respect the language in config
- change the `date_format` or `reported_date_format` and have a date in your message that should respect your format
- change `multipart_sms_limit` and have a super long message that should be truncated to the correct size 

There are other aspects of configuration that affect messages, but these are the most obvious ones. 
I did add an e2e test for this fix and it's testing with `locale_outgoing`.
